test/integration/dataContract/validateDataContractFactory.spec.js

```diff
@@ -760,6 +761,29 @@ describe('validateDataContractFactory', () => {
     expect(error.getIndexDefinition()).to.deep.equal(indexDefinition);
   });
 
```

Comment:
Do we need a test that we can use `$id` and `$userId` in indices?

Reply:
@shumkov updated a fixture, so positive test case checks it